site/monitoring-cluster-operator.md

```diff
@@ -0,0 +1,165 @@
+# Monitoring RabbitMQ in Kubernetes
+
+This topic describes how to monitor RabbitMQ instances deployed by Cluster Operator.
+
+## <a id='overview'></a> Overview
```

Comment:
We use headings that are links across the site. Not only this allows linking to a specific section, such headings visually stand out.